WeakAuras/BuffTrigger2.lua

```diff
@@ -261,9 +260,13 @@ local function UpdateToolTipDataInMatchData(matchData, time)
   end
   local changed = false
 
-  if matchData.unit and matchData.index and matchData.filter then
+  if matchData.unit and matchData.auraInstanceID then
```

Comment:
In you changes you basically passed in auraInstanceId as the index. I think that's strange and it's best to separate them.

And you missed the tooltip thing.

Reply:
for the tests, i wonder the impact the change on index field has on dynamic group sorting